common/builder/tests/testpystachemessagebuilder.py

```diff
@@ -0,0 +1,22 @@
+import os
```

Comment:
Just one on the file name. I'm not sure on best practice but I've been using underscore to separate words.

Reply:
As discussed offline, I think I was misinterpreting [PEP-8's section on package and module names](https://www.python.org/dev/peps/pep-0008/#package-and-module-names). It says pretty clearly that:
> Modules should have short, all-lowercase names. Underscores can be used in the module name if it improves readability.

I think underscores would improve readability here, so I'll make this change (and do the same in any similarly-named files).